app/__main__.py

```diff
@@ -27,4 +29,7 @@
 if not os.path.isfile(args["FILE_PATH"]):
     raise Exception("Given path does not lead to a file")
 
-run_core_functionality(args)
+img = load_image(args["FILE_PATH"], args["color_format"], args['resolution'])
+cv.imshow("Displayed photo", get_displayable(img))
```

Comment:
I know it's only temporary way of starting application, but the name of the opened file would be a good name for the window.